.github/workflows/clean-up-pr-caches.yml

```diff
@@ -0,0 +1,36 @@
+# https://docs.github.com/en/actions/using-workflows/caching-dependencies-to-speed-up-workflows#force-deleting-cache-entries
```

Comment:
Intersting to have especially in case of a full cache mode (yarn cache + node_modules + install-state). When a P/R is merged/closed.... it will free up the space explicitely.